### PR TITLE
Add support for WGS84

### PR DESCRIPTION
**Pull request by loicgasser**
ping @stebie @oterral 
 
So that we can try out, WGS84 and Mapbox GL via vector-forge as well. And WebMercator.

EPSG:4326 TMS (Cesium Globe)
EPSG:3857 (Google style minus the inversion of the origin, OSM style basically.)

### CODE REVIEW

File: vectorforge/lib/grids.py [hunk 1]
```diff
@@ -26,3 +27,246 @@ def getExtentAddress(self, zoom):
 
     def getResolution(self, zoom):
         return self.geodetic.Resolution(zoom)
+
+
+class GlobalMercator(object):
```
Comment (stebie) [inline vectorforge/lib/grids.py]: From where are these comments & code? Please mention the origin if it's not from you.

File: vectorforge/lib/grids.py [hunk 2]
```diff
@@ -26,3 +27,246 @@ def getExtentAddress(self, zoom):
...
+            quadKey += str(digit)
+
+        return quadKey
+
+
+class GlobalMercatorTileGrid:
+
+    def __init__(self, extent, tileSizePx=256.0):
+        self.extent = extent
+        self.geodetic = GlobalMercator(tileSize=tileSizePx)
```
Comment (stebie) [inline vectorforge/lib/grids.py]: self.geodetic is not appropriate for this projection. 

What about "globalMercator" instead?

File: README.md [hunk 3]
```diff
@@ -58,24 +58,25 @@ If debug parameter is provided, then multiprocessing is disabled.
...
+| minZoom                     | The minimal zoom where to start the tileing from             | 0                                          |
+| maxZoom                     | The maximal zoom where to stop the tileing at                | 26                                         |
+| gutter                      | The buffer in pixels around the tile                         | 100                                        |
+| isAnnotationLayer           | True if this is an annotation layer                          | false                                      |
+| maxFontSize                 | The maximum font size in pixels                              | 20                                         |
+| extent                      | The intersection extent selecting the tiles                  | [420000.0, 30000.0, 900000.0, 350000.0]    |
+| filters                     | A List of filters                                            | []                                         |
+| srid                        | A srid to determine the tiling scheme and output coordinates | 21781 (only additional support is 4326)    |
```
Comment (stebie) [inline README.md]: And with our additional commit it should actually read 

Description: A srid (EPGS code) to determine the tiling scheme and output coordinates. Supported projections: 21781 (CH1903 / LV03), 4326 (Global Geodetic) and 3857 (Global Mercator)

Default: 21781

**Comment by loicgasser** [comment]
I'll address your comments tomorrow. Thanks for the review

File: vectorforge/lib/grids.py (hunk 2 above)
Comment (loicgasser) [inline vectorforge/lib/grids.py]: yes I meant to change that. Makes sense.

File: vectorforge/lib/grids.py (hunk 1 above)
Comment (loicgasser) [inline vectorforge/lib/grids.py]: This part is from me.

Comment (loicgasser) [inline vectorforge/lib/grids.py]: The above kologan

File: vectorforge/lib/grids.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
+from quantized_mesh_tile.global_geodetic import GlobalGeodetic
+
+
+class TmsTileGrid:
```
Comment (loicgasser) [inline vectorforge/lib/grids.py]: Tile map service: https://en.wikipedia.org/wiki/Tile_Map_Service

File: README.md (hunk 3 above)
Comment (loicgasser) [inline README.md]: yes you're absolutely right.

File: configs/ch.swisstopo.swissnames3d_point.json [hunk 5]
```diff
@@ -0,0 +1,12 @@
+{
+  "layerBodId": "ch.swisstopo.swissnames3d_point",
+  "tileSizePx": 256,
+  "srid": 3857,
+  "extent": [5.86725126512748, 45.8026860136571, 10.9209100671547, 47.8661652478939],
```
Comment (loicgasser) [inline configs/ch.swisstopo.swissnames3d_point.json]: I may want to change those as well

File: vectorforge/lib/grids.py [hunk 6]
```diff
@@ -0,0 +1,272 @@
...
+        [maxRow, maxCol] = self.geodetic.LonLatToTile(
+            self.extent[2], self.extent[3], zoom
+        )
+        return [minRow, minCol, maxRow, maxCol]
+
+    def getResolution(self, zoom):
+        return self.geodetic.Resolution(zoom)
+
+
+class GlobalMercator(object):
```
Comment (loicgasser) [inline vectorforge/lib/grids.py]: As a reminder add source.

File: vectorforge/lib/grids.py [hunk 7]
```diff
@@ -0,0 +1,272 @@
...
+    def iterGrid(self, minZoom, maxZoom):
+        for zoom in xrange(minZoom, maxZoom + 1):
+            [minRow, minCol, maxRow, maxCol] = self.getExtentAddress(zoom)
+            for row in xrange(minRow, maxRow + 1):
+                for col in xrange(minCol, maxCol + 1):
+                    tileBounds = self.geodetic.TileBounds(row, col, zoom)
+                    yield (tileBounds, zoom, col, row)
+
+    def getExtentAddress(self, zoom):
+        [minX, minY] = self.geodetic.LatLonToMeters(
```
Comment (loicgasser) [inline vectorforge/lib/grids.py]: change this if coordinates are already in meters

**Comment by loicgasser** [comment]
changes have been applied

**Comment by loicgasser** [comment]
@stebie here is a code snippet using swissnames3d points

http://codepen.io/loicgasser/pen/XpBJWJ?editors=0010

We have 223'818 points in total.
Those are all available at all zoom levels so it's very lagy. (and all attributes are in it)
I'll filter those according to Cédric and Kirsten input.

**Pull request merged**